assets/mkEnv/set-prompt.sh

```diff
@@ -0,0 +1,36 @@
+# Tweak the (already customized) prompt: add a flox indicator.
+
+LIGHTBLUE256=61 # SlateBlue3
+DARKPEACH256=216 # LightSalmon1
+FLOX_PROMPT_COLOR_1=${FLOX_PROMPT_COLOR_1:-$LIGHTBLUE256}
+FLOX_PROMPT_COLOR_2=${FLOX_PROMPT_COLOR_2:-$DARKPEACH256}
+
+_esc="\x1b["
+colorReset="\[${_esc}0m\]"
+colorBold="\[${_esc}1m\]"
+colorPrompt1="\[${_esc}38;5;${FLOX_PROMPT_COLOR_1}m\]"
+colorPrompt2="\[${_esc}38;5;${FLOX_PROMPT_COLOR_2}m\]"
+_floxPrompt1="${colorPrompt1}flox"
+_floxPrompt2="${colorPrompt2}[$FLOX_PROMPT_ENVIRONMENTS]"
+_flox=$(echo -e -n "${colorBold}${FLOX_PROMPT-$_floxPrompt1} ${_floxPrompt2}${colorReset} ")
+unset _esc colorReset colorBold colorPrompt1 colorPrompt2 _floxPrompt1 _floxPrompt2
+
+if [ -n "$_flox" -a -n "${PS1:-}" ]
```

Comment:
blocking change request: Effects `[` compatibility across BSD/GNU and various shells which may or may not have `[` as a builtin.

```suggestion
if [ -n "$_flox" ] && [ -n "${PS1:-}" ]
```